doc/user_guide/howto_create_new_dialect.md

```diff
@@ -0,0 +1,109 @@
+# How to Create a new Dialect of the Document Files Virtual Schema
```

Comment:
It's much better now :+1: 
But rename a file itself please - > `dialect_development_guide` o r something similar

Reply:
I will do that later when we resolved the other conversations since they could get outdated by that.